index.rst

```diff
@@ -50,7 +50,696 @@
+| UNIQUE (name, camera_id)                                                |
++-------------------------------------------------------------------------+
+
+PhysicalSensors actually represent the "slot" for a sensor in a camera,
+independent of both any observations and the actual detector (which may change
+over the life of the camera).  The ``group`` field may mean different things
+for different cameras (such as rafts for LSST, or groups of sensors oriented
+the same way relative to the focal plane for HSC).  The ``purpose`` field
+indicates the role of the sensor (such as science, wavefront, or guiding).
+Becaues some cameras identify sensors with string names and other use numbers,
+we provide fields for both; the name may be a stringified integer, and the
+number may be autoincrement.
+
++----------------------+----------+-----------------------------------------------------------+
+| *Visit*                                                                                     |
++======================+==========+===========================================================+
+| visit_id             | uint64   | PRIMARY KEY                                               |
++----------------------+----------+-----------------------------------------------------------+
+| number               | uint64   | NOTNULL                                                   |
++----------------------+----------+-----------------------------------------------------------+
+| camera_id            | uint64   | NOT NULL REFERENCES Camera (camera_id)                    |
++----------------------+----------+-----------------------------------------------------------+
+| physical_filter_id   | uint64   | NOT NULL REFERENCES AbstractFilter (abstract_filter_id)   |
++----------------------+----------+-----------------------------------------------------------+
+| obs_begin            | datetime | NOT NULL                                                  |
++----------------------+----------+-----------------------------------------------------------+
+| obs_end              | datetime | NOT NULL                                                  |
++----------------------+----------+-----------------------------------------------------------+
+| region               | blob     |                                                           |
++----------------------+----------+-----------------------------------------------------------+
+| UNIQUE (number, camera_id)                                                                  |
++---------------------------------------------------------------------------------------------+
+
+Entries in the Visit table correspond to observations with the full camera at
+a particular pointing, possibly comprised of multiple exposures (Snaps).  A
+Visit's ``region`` field holds an approximate but inclusive representation of
+its position on the sky that can be compared to the ``regions`` of other
```

Comment:
Compared how? I guess not in the database because it is a `blob`. Won't that complicate spatial queries?

Reply:
We also have an in-database join table that holds all of that overlap information, which may need to have its entries generated outside the database.  It's possible that some databases will allow this to be a non-blob type that could be used to generate that join table in the database or even make it a view, but for the purposes of the common schema it's considered opaque.